### DISCUSSION TITLE
Allow the addition of extra frontmatter fields to event notes without breaking notes

### DISCUSSION THREAD

**Discussion by GabrielleVJ** [davish/obsidian-full-calendar#439] [Feature Requests]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/davish/obsidian-full-calendar/discussions/categories/feature-requests) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/davish/obsidian-full-calendar/issues) didn't find any similar feature requests.

### 🔖 Feature description

I would like to add additional front matter to full note events, without this breaking the note and removing it from the calendar.

### ✔️ Solution

For example, I like to have an ‘attendee:’ field in my meeting notes, and a ‘project:’ field so I can link the meet to its relevant projects. I’m happy to manually add these fields after the standard meeting note is created, but this is not currently possible as any changes to the front matter break the events.

**Comment by davish** [comment]
Hi @GabrielleVJ, how does the event "break" when you add new frontmatter? I'm pretty sure other frontmatter fields should be ignored by the plugin.
